drake/systems/plants/rigidBodyTreeMexFunctions.cpp

```diff
@@ -188,3 +188,29 @@ void dynamicsBiasTermmex(int nlhs, mxArray *plhs[], int nrhs, const mxArray *prh
 
   mexTryToCallFunctions(nlhs, plhs, nrhs, prhs, true, func_double, func_autodiff_fixed_max, func_autodiff_dynamic);
 }
+
+template <typename Scalar>
+Eigen::Matrix<Scalar, Eigen::Dynamic, Eigen::Dynamic> velocityToPositionDotMapping(const KinematicsCache<Scalar>& cache) {
+  auto nq = cache.getNumPositions();
+  return cache.transformPositionDotMappingToVelocityMapping(Matrix<Scalar, Eigen::Dynamic, Eigen::Dynamic>::Identity(nq, nq));
```

Comment:
i'm confused.  why is the vToqdot function calling transformPositionDotToVelocityMapping?
(and vice versa below)?  either need really clear doc or better names.